### PR TITLE
Create jest-test.yml

### PR DESCRIPTION
**Pull request by Killer2OP**
## Related Issue

Added a github action to run the jest test file

Closes: 

## Checklist:

- [x] My code adheres to the established style guidelines of this project.
- [x] I have conducted a self-review of my code.
- [x] I have included comments in areas that may be difficult to understand.
- [x] I have made corresponding updates to the project documentation.
- [x] My changes have not introduced any new warnings.

## Screenshots
image

### CODE REVIEW

**Comment by Killer2OP** [comment]
@anmode

File: .github/workflows/jest-test.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```
Comment (anmode) [inline .github/workflows/jest-test.yml:20]: can we do it 16?

**Review by anmode — CHANGES_REQUESTED** [review]

File: .github/workflows/jest-test.yml (hunk 1 above)
Comment (Killer2OP) [reply .github/workflows/jest-test.yml:20]: okie

**Comment by Killer2OP** [comment]
@anmode DONE.

**Pull request merged**